spec/monads/many_spec.rb

```diff
@@ -126,5 +126,31 @@ module Monads
         end
       end
     end
+
+    describe "unpacking nested many values" do
+      let(:expected_values) do
+        2.times.map do |i|
```

Comment:
Do you think it is reasonable to use so many #map, #times and #flatten, just to introduce values we are expecting?
I think it would be more readable and useful (for understanding purpose) if you'll define simple arrays. 
Just like in your issue.
WDYT?